token/services/network/orion/views.go

```diff
@@ -16,19 +18,33 @@ type ResponderRegistry interface {
 
 func InstallViews(viewRegistry ResponderRegistry, dbManager *DBManager, statusCache TxStatusResponseCache) error {
 	logger.Debugf("Installing custodian views...")
+	keyTranslator := &translator.HashedKeyTranslator{KT: &keys.Translator{}}
+
 	if err := viewRegistry.RegisterResponder(&PublicParamsRequestResponderView{}, &PublicParamsRequestView{}); err != nil {
 		return err
 	}
-	if err := viewRegistry.RegisterResponder(&RequestApprovalResponderView{dbManager: dbManager, statusCache: statusCache}, &RequestApprovalView{}); err != nil {
+	if err := viewRegistry.RegisterResponder(&RequestApprovalResponderView{
+		dbManager:     dbManager,
```

Comment:
Shouldn't these services be injected maybe instead of passed as params?

Reply:
We can refactor this in another PR